Competition2017/tests/competition/subsystems/collector/CollectorTestBase.java

```diff
@@ -13,7 +13,7 @@ public void setUp() {
         
         collect = injector.getInstance(TestCollectorSubsystem.class);
     }
-    
+    //Tests to see if the motor is running at the given power.
```

Comment:
Hint: type `/**` and hit enter. The comment that is generated is called a "Javadoc" comment; it is detected specially by Eclipse and shows a caller the description you provide.